doc/source/rllib-training.rst

```diff
@@ -140,6 +140,14 @@ Here is an example of the basic usage (for a more complete example, see `custom_
            checkpoint = trainer.save()
            print("checkpoint saved at", checkpoint)
 
+    # Also, in case you have trained a model outside of ray/RLlib and have created
+    # a h5-file with weight values in it, you can load it into your Trainer's
+    # model (tf or torch) by doing:
+    trainer.import_model("my_weights.h5")
```

Comment:
Could you add a small example, or link to the Keras docs on creating these h5 files? This is so users not familiar with Keras know what this is referring to (i.e., me :)).

Reply:
Done.